units/host_termination.go

```diff
@@ -391,6 +392,21 @@ func (j *hostTerminationJob) Run(ctx context.Context) {
 			}
 		}
 	}
+
+	provisioningFailedStatuses := []string{evergreen.HostStarting, evergreen.HostProvisioning, evergreen.HostProvisionFailed, evergreen.HostBuilding}
```

Comment:
Put this list in globals https://github.com/evergreen-ci/evergreen/blob/c4b7e18d71f1e3cb0f3a8a2bf74fd65935f21fde/globals.go#L427